src/validation/isGitAvailable.ts

```diff
@@ -1,8 +1,8 @@
-import { findGitRoot } from '../paths';
+import { findProjectRoot } from '../paths';
 import { git } from '../git';
 
 export function isGitAvailable(cwd: string) {
   const result = git(['--version']);
-  const gitRoot = findGitRoot(cwd);
+  const gitRoot = findProjectRoot(cwd);
```

Comment:
This should still be "findGitRoot" because the function here is trying to verify that .git exists:

```suggestion
  const gitRoot = findGitRoot(cwd);
```

Reply:
Thanks! Good catch 😅